admin/coupon_restrict.php

```diff
@@ -500,7 +496,7 @@
 }
 unset($products);
 ?>
-            <tr class="smallText">
+            <tr class="smallText" colspan="2">
```

Comment:
Are you sure about this? colspan on a tr?

Reply:
No, should not be there. Good spot! Will remove retest and reissue.